README.md

```diff
@@ -1,268 +1,334 @@
-
-    // Saml Form Template Location
-    'saml-form-template.njk',
-    // This is an optional parameter which can be used to style the saml form
-    // used to send the authn request to Verify.
-    // This template should only be rendered if Javascript has been disabled in the user's browser.
-    // If provided, the ssoLocation and samlRequest recieved from the Verify Service Provider
-    // will be provided to the named template for rendering.
-    // If this is not provided, passport-verify will render a default auto posting form
-    // with the correct attributes.
-
-    // Level of Assurance
-    // LEVEL_1 or LEVEL2 depending on your service's requirements. Defaults to LEVEL_2.
-    'LEVEL_2'
-   ))
-   ```
-
-1. Configure routes for the authentication flow
-
-   See [createResponseHandler](https://alphagov.github.io/passport-verify/modules/_create_response_handler_.html#createresponsehandler)
-   and its [callbacks](https://alphagov.github.io/passport-verify/interfaces/_create_response_handler_.responsescenarios.html#onauthnfailed) in the API documentation.
-
-   ```javascript
-   // route for authenticating a user
-   app.post('/verify/start', passport.authenticate('verify'))
-
-   // route for handling a callback from verify
-   app.post('/verify/response', (req, res, next) => (
-   
-     // in this example, authenticate() is being called from within the route handler
-     // rather than being used as middleware, this provides access to the request
-     // and response objects through closure
-     const authMiddleware = passport.authenticate('verify', function (error, user, infoOrError, status) {
-
-      if (error) {
-        return res.send(`TODO: render error-page with message ${error: error.message}`)
-      }
-
-      if (user) {
-        // passport-verify requires the use of a custom callback to handle successful
-        // authentication
-        return req.logIn(user, () => res.send('TODO: redirect to service landing page')))
-      }
-
-      return res.send(`TODO: redirect to authentication failed page with ${error: infoOrError}`)
-
-    })
-    authMiddleware(req, res, next)
-   ```
-
-   See [the example implementation](https://github.com/alphagov/passport-verify-stub-relying-party/blob/master/src/app.ts) for
-   a more detailed example with session support.
-
-Logging
------------
+`passport-verify` is a node and [passport.js](http://passportjs.org/) client for the [Verify Service Provider (VSP)](https://github.com/alphagov/verify-service-provider).
+
+## Before you start
+
+[Set up the VSP](https://github.com/alphagov/verify-service-provider/README.md).
+
+## Usage
+
+### Step 1. Install`passport-verify`
+
+```bash
+npm install --save passport-verify
+```
+
+### Step 2. Configure the `passport-verify` strategy
+
+**TODO** To configure the strategy you need the info in the code blocks when following instructions on here => [Configure `passport-verify` strategy](http://passportjs.org/docs/configure#strategies).
```

Comment:
Andrea, you were planning to return to this, right?

Reply:
yup, I've addressed it in the subsequent commit 👌 

it was only later when I realised what the TODOs in the examples were about - it's where users need to fill in details!... This is not common practice in my (limited) experience, but happy for them to stay in if it's common in the Node.js community.